velox/type/ShortDecimal.h

```diff
@@ -25,7 +25,8 @@ namespace facebook::velox {
 
 struct ShortDecimal {
  public:
-  explicit ShortDecimal(int64_t value) : unscaledValue_(value) {}
+  constexpr ShortDecimal() : unscaledValue_(0) {}
```

Comment:
Where do we need this?

Reply:
Some vector initialization required a default constructor. The compilation was failing.